WireAnalytics/Package.swift

```diff
@@ -19,20 +22,16 @@ let package = Package(
     targets: [
         .target(
             name: "WireAnalytics",
-            dependencies: resolveWireAnalyticsDependencies() + [
+            dependencies: [
                 .product(name: "Countly", package: "countly-sdk-ios")
             ],
             swiftSettings: swiftSettings
         ),
         .target(
             name: "WireDatadog",
-            dependencies: [
-                .product(name: "DatadogCore", package: "dd-sdk-ios"),
-                .product(name: "DatadogCrashReporting", package: "dd-sdk-ios"),
-                .product(name: "DatadogLogs", package: "dd-sdk-ios"),
-                .product(name: "DatadogRUM", package: "dd-sdk-ios"),
-                .product(name: "DatadogTrace", package: "dd-sdk-ios")
-            ],
+            dependencies: datadogDependencies(),
+            path: "Sources/WireDatadog",
```

Comment:
Did you consider making the path change instead of the source files? It might be easier to maintain 🤔

Reply:
from the doc it was saying here lies the directory and on the other one just filenames. I didn't play with this